app/Http/Transformers/ReactionTransformer.php

```diff
@@ -18,6 +18,15 @@ public function transform(Reaction $reaction)
         return [
             'northstar_id' => $reaction->northstar_id,
             'post_id' => (string) $reaction->post_id,
+            'term' => [
+                'name' => 'heart',
+                'id' => 641,
+                'total' => 1,
+            ],
+            'current_user' => [
```

Comment:
Sorry I'm still confused. The `northstar_id` above it is the id of the user who did the reaction, so Why do we need to also return this? I think I was confused because I understand why we return it in the `ReportbackTransformer` but not here.